packages/gatsby/src/schema/infer/inference-metadata.ts

```diff
@@ -509,7 +509,7 @@ const initialMetadata = (state?: object): ITypeMetadata => {
     total: 0,
     ignoredFields: undefined,
     fieldMap: {},
-    ...state,
+    ...state
```

Comment:
Please drop changes in this file (maybe update your Prettier or figure out why the git commit hooks don't run)